shopinvader/services/customer.py

```diff
@@ -45,11 +45,19 @@ def _validator_create(self):
                 "email": {"type": "string", "required": True},
                 "external_id": {"type": "string", "required": True},
                 "vat": {"type": "string", "required": False},
+                "lang": {"type": "string", "required": False},
+                "company_name": {"type": "string", "required": False},
             }
         )
         return schema
 
+    def _empty_fields_to_drop(self):
+        return ["lang", "company_name"]
+
     def _prepare_params(self, params):
+        for key in self._empty_fields_to_drop():
```

Comment:
@simahawk I don't understand why empty fields should be dropped. The `required` attribute into the cerberus scheme means that the field must be (or not) present into the json whatever their values. The attribute `nullable`can be used to say if a field can be set to null....This allows the following usages:
`{'my_field': null}`  -> reset the value to null for `my_field`
`{}`  -> nothing to change for my field
`{'my_field': 'val1'l}` -> set value `'val1'` for `my_field`

Reply:
If you don't want to modify a field, don't put it into the json...

Reply:
but the client might send it anyway (eg: an hidden field). I just want to be defensive in such cases. :man_shrugging:

Reply:
it's the responsibility of the client to sent or not the field. Otherwise how will it be possible to reset these fields?

Reply:
I removed the hack, in any case if we have issues we can think about it later. Thanks for your feedback :)